### PR TITLE
feat: add translate at the homepage

### PR DESCRIPTION
**Pull request by BLOCKCHAINDEVWEB**
I need to change the dimension of the texts or to adapt the text to the cell and this with the French and English language.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -31,10 +31,10 @@ const HeroSection: React.FC = () => {
       <div className="mx-auto px-4 py-32 lg:-mt-[90px] lg:flex lg:h-screen lg:items-center">
         <div className="mx-auto -mt-12 text-center">
           <h1 className="hubot-sans text-4xl font-black tracking-tight text-gray-900 sm:text-6xl">
-            WEB3 DEVTOOL
+          {translate('title_h1')}
```
Comment (n1c01a5) [inline pages/index.tsx:34]: Use something more explicit like `title_main`.

File: pages/index.tsx [hunk 2]
```diff
@@ -31,10 +31,10 @@ const HeroSection: React.FC = () => {
...
           </h1>
           <h2 className="mt-1 font-inter text-xs font-extralight leading-8 text-black sm:text-sm lg:text-lg">
-            🏁The Fatest Way to Fix Github Issues 🐙 for Open Source Project 🦄
+            {translate('title_h2')}
```
Comment (n1c01a5) [inline pages/index.tsx:37]: Use something more explicit like `subtitle_main`.

File: public/locales/fr/homepage.json [hunk 3]
```diff
@@ -1,51 +1,68 @@
 {
-	"title": "Incitez les développeurs à réaliser vos issues Github",
-	"description": "web3 freelancing",
-	"hero_heading_part1": "Incitez les développeurs",
-	"hero_heading_part2": "à réaliser vos Issues Github",
-	"hero_subheading": "Attachez une récompense à vos fonctionnalités, corrections et tests,\npour inciter les développeurs à les réaliser dans le temps imparti.",
-	"hero_try_free": "Essayer Gratuitement",
-	"hero_button_book_demo": "Réserver une démo",
+  "title_h1": "OUTIL DE TRAVAIL WEB3",
```
Comment (n1c01a5) [inline public/locales/fr/homepage.json:2]: Keep `DEVTOOL WEB3`.

File: public/locales/fr/homepage.json [hunk 4]
```diff
@@ -1,51 +1,68 @@
...
+  "build_faster_text1_row1_section": "Accorder une aide économique 🤑",
+  "build_faster_text1_row2_section": "Incitations à la contribution",
+  "build_faster_text2_row1_section": "L'examen du code",
+  "build_faster_text2_row2_section": "Par la communauté 🧐",
+  "build_faster_text3_row1_section": "Payer les cotisants",
+  "build_faster_text3_row2_section": "Automatiquement 🤖",
+  "build_cheaper_title_section": "CONSTRUIRE MOINS CHER",
+  "build_cheaper_text_row1_section": "Bénéficier de 0 % de frais 😎",
+  "build_cheaper_text_row2_section": "Transaction de pair à pair",
+  "build_trustless_title_section": "CONSTRUIRE SANS CONFIANCE",
```
Comment (n1c01a5) [inline public/locales/fr/homepage.json:14]: "build_trustless_title_section": "+ CONFIANCE"

File: public/locales/fr/homepage.json [hunk 5]
```diff
@@ -1,51 +1,68 @@
...
-	"title": "Incitez les développeurs à réaliser vos issues Github",
-	"description": "web3 freelancing",
-	"hero_heading_part1": "Incitez les développeurs",
-	"hero_heading_part2": "à réaliser vos Issues Github",
-	"hero_subheading": "Attachez une récompense à vos fonctionnalités, corrections et tests,\npour inciter les développeurs à les réaliser dans le temps imparti.",
-	"hero_try_free": "Essayer Gratuitement",
-	"hero_button_book_demo": "Réserver une démo",
+  "title_h1": "OUTIL DE TRAVAIL WEB3",
+  "title_h2": "🏁La manière la plus rapide de résoudre les problèmes sur Github 🐙 pour les projets Open Source 🦄",
+  "build_faster_title_section": "CONSTRUIRE PLUS VITE",
```
Comment (n1c01a5) [inline public/locales/fr/homepage.json:4]: "build_faster_title_section": "+ RAPIDE",

File: public/locales/fr/homepage.json [hunk 6]
```diff
@@ -1,51 +1,68 @@
...
+  "build_faster_text1_row1_section": "Accorder une aide économique 🤑",
+  "build_faster_text1_row2_section": "Incitations à la contribution",
+  "build_faster_text2_row1_section": "L'examen du code",
+  "build_faster_text2_row2_section": "Par la communauté 🧐",
+  "build_faster_text3_row1_section": "Payer les cotisants",
```
Comment (n1c01a5) [inline public/locales/fr/homepage.json:9]: "build_faster_text3_row1_section": "Payer les contributeurs",

File: public/locales/fr/homepage.json [hunk 7]
```diff
@@ -1,51 +1,68 @@
...
-	"features_accounting_description": "La comptabilité est souvent une tâche pénible, d'autant plus s'il s'agit de cryptomonnaies, qui sont souvent très volatiles. Une interface simple calcule l'équivalent en euros et en dollars pour vous simplifier la vie.",
+  "features_performance_heading": "Performance",
+  "features_performance_description": "Grâce à un tableau de bord qui suit les performances de chaque développeur en temps réel, vous pouvez directement faire un retour, permettant au développeur de réaliser une tâche plus efficacement.",
 
-	"features_more": "Plus de fonctionnalités",
+  "features_accounting_heading": "Comptabilité",
+  "features_accounting_description": "La comptabilité est souvent une tâche pénible, d'autant plus s'il s'agit de cryptomonnaies, qui sont souvent très volatiles. Une interface simple calcule l'équivalent en euros et en dollars pour vous simplifier la vie.",
 
-	"testimonial_at": "à",
+  "features_more": "Plus de fonctionnalités",
```
Comment (n1c01a5) [inline public/locales/fr/homepage.json:55]: "features_more": "Fonctionnalités",

File: public/locales/fr/homepage.json [hunk 8]
```diff
@@ -1,51 +1,68 @@
...
+  "title_h1": "OUTIL DE TRAVAIL WEB3",
+  "title_h2": "🏁La manière la plus rapide de résoudre les problèmes sur Github 🐙 pour les projets Open Source 🦄",
+  "build_faster_title_section": "CONSTRUIRE PLUS VITE",
+  "build_faster_text1_row1_section": "Accorder une aide économique 🤑",
+  "build_faster_text1_row2_section": "Incitations à la contribution",
+  "build_faster_text2_row1_section": "L'examen du code",
+  "build_faster_text2_row2_section": "Par la communauté 🧐",
+  "build_faster_text3_row1_section": "Payer les cotisants",
+  "build_faster_text3_row2_section": "Automatiquement 🤖",
+  "build_cheaper_title_section": "CONSTRUIRE MOINS CHER",
```
Comment (n1c01a5) [inline public/locales/fr/homepage.json:11]: "build_cheaper_title_section": "- CHER",

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (BLOCKCHAINDEVWEB) [reply pages/index.tsx]: ok

File: pages/index.tsx (hunk 2 above)
Comment (BLOCKCHAINDEVWEB) [reply pages/index.tsx]: ok

File: public/locales/fr/homepage.json (hunk 3 above)
Comment (BLOCKCHAINDEVWEB) [reply public/locales/fr/homepage.json]: ok

File: public/locales/fr/homepage.json (hunk 5 above)
Comment (BLOCKCHAINDEVWEB) [reply public/locales/fr/homepage.json]: ok

File: public/locales/fr/homepage.json (hunk 6 above)
Comment (BLOCKCHAINDEVWEB) [reply public/locales/fr/homepage.json]: ok

File: public/locales/fr/homepage.json (hunk 8 above)
Comment (BLOCKCHAINDEVWEB) [reply public/locales/fr/homepage.json]: ok

File: public/locales/fr/homepage.json (hunk 4 above)
Comment (BLOCKCHAINDEVWEB) [reply public/locales/fr/homepage.json]: ok

File: public/locales/fr/homepage.json (hunk 7 above)
Comment (BLOCKCHAINDEVWEB) [reply public/locales/fr/homepage.json]: ok

File: public/locales/fr/footer.json [hunk 9]
```diff
@@ -1,7 +1,18 @@
...
-	"language_french": "Français",
-	"language_english": "Anglais",
+  "footer_solutions_title": "Solutions",
+  "footer_solutions_name_row1": "Analytiques",
+  "footer_support_title": "Support",
+  "footer_support_name_row1": "Documentation",
+  "footer_support_name_row2": "Aide",
+  "footer_company_title": "Entreprise",
+  "footer_company_name_row1": "Analyse",
+  "footer_company_name_row2": "A propos",
```
Comment (n1c01a5) [inline public/locales/fr/footer.json:9]: À propos

File: public/locales/fr/homepage.json [hunk 10]
```diff
@@ -1,51 +1,68 @@
 {
-	"title": "Incitez les développeurs à réaliser vos issues Github",
-	"description": "web3 freelancing",
-	"hero_heading_part1": "Incitez les développeurs",
-	"hero_heading_part2": "à réaliser vos Issues Github",
-	"hero_subheading": "Attachez une récompense à vos fonctionnalités, corrections et tests,\npour inciter les développeurs à les réaliser dans le temps imparti.",
-	"hero_try_free": "Essayer Gratuitement",
-	"hero_button_book_demo": "Réserver une démo",
+  "title_main": "DEVTOOL WEB3",
+  "subtitle_main": "🏁La manière la plus rapide de résoudre les Issues sur Github 🐙 pour les projets Open Source 🦄",
```
Comment (n1c01a5) [inline public/locales/fr/homepage.json:3]: Issues Github

Comment (BLOCKCHAINDEVWEB) [reply public/locales/fr/homepage.json]: ok

File: public/locales/fr/footer.json (hunk 9 above)
Comment (BLOCKCHAINDEVWEB) [reply public/locales/fr/footer.json]: ok

**Review by n1c01a5 — APPROVED** [review]

**Pull request merged**